instrumentation/grpc-1.6/testing/src/main/groovy/io/opentelemetry/instrumentation/grpc/v1_6/AbstractGrpcTest.groovy

```diff
@@ -276,15 +288,15 @@ abstract class AbstractGrpcTest extends InstrumentationSpecification {
               "message.id" 1
             }
           }
-          if (grpcStatus.cause != null) {
-            errorEvent grpcStatus.cause.class, grpcStatus.cause.message, 1
-          }
+          errorEvent grpcStatus.asRuntimeException().class, grpcStatus.asRuntimeException().message, 1
```

Comment:
Note the change in telemetry here. But I think it does make sense to record the StatusRuntimeException (otherwise user could just throw a normal RuntimeException)